raiden/utils/migrations/v18_to_v19.py

```diff
@@ -0,0 +1,45 @@
+import json
+
+from web3 import Web3
+
+from raiden.storage.sqlite import SQLiteStorage, StateChangeRecord
+
+SOURCE_VERSION = 18
+TARGET_VERSION = 19
+
+
+def _add_blockhash_to_state_changes(storage: SQLiteStorage, web3: Web3) -> None:
+    """Adds blockhash to ContractReceiveXXX and ActionInitChain state changes"""
+    state_changes = storage.get_all_state_changes()
+    updated_state_changes = []
+    for state_change in state_changes:
+        data = json.loads(state_change.data)
+        affected_state_change = (
+            'raiden.transfer.state_change.ContractReceive' in data['_type'] or
+            'raiden.transfer.state_change.ActionInitChain' in data['_type']
+        )
+        if affected_state_change:
+            assert 'block_hash' not in data, 'v18 state changes cant contain blockhash'
```

Comment:
maybe a "can not" to be gramatically correct

Reply:
sorry I just noticed it is still WIP

Reply:
We are also limited by line length :)